scripts/run-android.sh

```diff
@@ -0,0 +1,62 @@
+#!/usr/bin/env bash
+set -euo pipefail
+set -m # needed to access jobs
+
+export BUILD_ENV=debug
+export BUILD_TYPE=debug
+GIT_ROOT=$(cd "${BASH_SOURCE%/*}" && git rev-parse --show-toplevel)
+
+# generates android debug build when we pass the debug flag
+"${GIT_ROOT}"/scripts/build-android.sh
+
+# 25 seconds is enough for metro to start, if it doesn't we have problems
+TIMEOUT=25
+TIMESPENT=1
```

Comment:
There is no need for two variables, you can just decrement `TIMEOUT` until it's 0.

Reply:
addressed here : https://github.com/status-im/status-mobile/pull/18784/commits/d6a19d75da1010ab9e33dd395ba4c6e45e7456e6